sudo/src/helpers.rs

```diff
@@ -305,7 +273,11 @@ pub fn env_as_string() -> String {
 
         // Try to figure out the end of the double-null terminated env block.
         loop {
-            let len = wcslen(PCWSTR(end));
+            extern "C" {
+                fn wcslen(s: *const u16) -> usize;
+            }
+
+            let len = wcslen(end);
```

Comment:
Sudo was depending on a hidden internal import of `wcslen` from the `windows-core` crate that no longer exists. The solution is just to define this CRT import directly.